Update XMCL install/export help (#47)
modrinth/docs: docs/modpacks/playing_modpacks.md

## How do I play a modpack?

> Currently, the easiest ways to play a Modrinth modpack are via [ATLauncher](https://atlauncher.com), [BakaXL](http://jk-insider.bakaxl.com:8888), [XMCL](https://xmcl.app), [MultiMC](https://multimc.org), or [PolyMC](https://polymc.org). You can also install Modrinth packs if you switch to the development branch of [Hello Minecraft! Launcher (HMCL)](https://github.com/huanghongxun/HMCL).

Reviewer: These are not in alphabetical order

Author: Done.

![MultiMC](../../static/img/tutorial/multimc.png)

If you want to insert a URL directly, you can do so by selecting "Import from zip" instead of "Modrinth".

### X Minecraft Launcher (XMCL)

> X Minecraft Launcher (XMCL) is an open source Minecraft launcher manages the mods & other resources via instances model similar to MultiMC.

Reviewer 2: ```suggestion
X Minecraft Launcher (XMCL) is an open source Minecraft launcher which provides a disk efficient way to manage massive mods and other resources similarly to MultiMC.
```

You can find the Modrinth tab on the side bar of the Launcher.

> You need to switch the Mod to Modpack in the dropdown on the top:

Reviewer 2: ```suggestion
To download modpacks, switch to Modpack in the dropdown on the top:
```

![Home](../../static/img/tutorial/xmcl-modrinth-home.png)

> Click the project card and go to the `VERSIONS` tab. Click download button in the `VERSIONS` tab, or click `Install` button on the right to install.

Reviewer 2: ```suggestion
Click the project card and go to the `Versions` tab. Click the download button in the `Versions` tab or click the `Install` button on the right.
```

![Project](../../static/img/tutorial/xmcl-modrinth-project.png)

> Once the download is finished, it will show a snackbar asking you to create a new instance. Click the `>` button to open the creation dialog.

Reviewer 2: ```suggestion
Once the download is finished, ask whether you want to create a new instance. Click the `>` button to open the creation dialog.
```
